### PR TITLE
email.mime: add policy arguments

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

File: stdlib/3/email/mime/text.pyi [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Stubs for email.mime.text (Python 3.4)
 
-from typing import Optional
 from email.mime.nonmultipart import MIMENonMultipart
+from typing import Optional
 
 class MIMEText(MIMENonMultipart):
-    def __init__(self, _text: str, _subtype: str = ...,
-                 _charset: Optional[str] = ...) -> None: ...
+    def __init__(self, _text: str, _subtype: str = ..., _charset: Optional[str] = ...) -> None: ...
```
Comment (srittau) [inline stdlib/3/email/mime/text.pyi]: Did you miss it here?

Comment (hauntsaninja) [reply stdlib/3/email/mime/text.pyi]: Thanks, good catch. Missed it in another place as well :-)

**Comment by JelleZijlstra** [comment]
CI is failing (a flake8 issue and some now-unused whitelist entries).

File: stdlib/3/email/mime/text.pyi (hunk 1 above)
Comment (hauntsaninja) [reply stdlib/3/email/mime/text.pyi]: Looks like these are issues which stubtest already caught before #3835. I was fixing the delta, which is why I missed them.

**Pull request merged**